net/mwan3/files/etc/config/mwan3

```diff
@@ -142,4 +142,9 @@ config rule 'https'
 config rule 'default_rule'
 	option dest_ip '0.0.0.0/0'
 	option use_policy 'balanced'
+	option family 'ipv4'
 
+config rule 'default_rule'
+	option dest_ip '::'
+	option use_policy 'balanced'
+	option family 'ipv6'
```

Comment:
This should go to a separate commit, because we change the default configuration

Reply:
Shouldn't the destination IP for the default_rule_v6 rule be `option dest_ip '::/0'`? With just `::` I see no matches on the Active ipv6 user rule, but with ::/0 there are matches.